libfeed/feed-text-sanitization.c

```diff
@@ -0,0 +1,118 @@
+/* Copyright 2018 Endless Mobile, Inc.
+ *
+ * eos-discovery-feed is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * eos-discovery-feed is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with eos-discovery-feed.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+#include <string.h>
+
+#include <glib.h>
+
+#include "feed-text-sanitization.h"
```

Comment:
Should include `config.h`

Reply:
I don't think we generate one at the moment. I'll do that now.